notebooks/D1-entitlement-demographics.ipynb

```diff
@@ -322,110 +340,381 @@
+       "4  06037101110               46           1927             0.023871   \n",
+       "\n",
+       "   zero_veh_workers  pct_zero_veh_workers  pop_renter  pop_total  \\\n",
+       "0                 0                   0.0        2199       4219   \n",
+       "1                 0                   0.0        2199       4219   \n",
+       "2                 0                   0.0        2199       4219   \n",
+       "3                 0                   0.0        2199       4219   \n",
+       "4                 0                   0.0        2199       4219   \n",
+       "\n",
+       "   pct_pop_renter  pop_whitenonhisp  ...  total_r75to99  total_r100to124  \\\n",
+       "0        0.521214              2516  ...            277               91   \n",
+       "1        0.521214              2516  ...            277               91   \n",
+       "2        0.521214              2516  ...            277               91   \n",
+       "3        0.521214              2516  ...            277               91   \n",
+       "4        0.521214              2516  ...            277               91   \n",
+       "\n",
+       "   total_r125to149  total_r150to199  total_gt200  total_total      density  \\\n",
+       "0               93              109           44         1596  9565.898824   \n",
+       "1               93              109           44         1596  9565.898824   \n",
+       "2               93              109           44         1596  9565.898824   \n",
+       "3               93              109           44         1596  9565.898824   \n",
+       "4               93              109           44         1596  9565.898824   \n",
+       "\n",
+       "   year  suffix  count  \n",
+       "0  2006      AA      1  \n",
+       "1  2007      AA      1  \n",
+       "2  2014      AA      1  \n",
+       "3  2017      AA      1  \n",
+       "4  2007   APCNV      1  \n",
+       "\n",
+       "[5 rows x 33 columns]"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# Merge the income data with the entitlements counts:\n",
+    "# Merge the census data with the entitlements counts:\n",
     "joined = pandas.merge(\n",
-    "    census.set_index(\"GEOID\"),\n",
+    "    census,\n",
     "    entitlement,\n",
-    "    how=\"left\",\n",
-    "    left_index=True,\n",
-    "    right_index=True,\n",
+    "    on=\"GEOID\",\n",
+    "    how=\"left\", \n",
+    "    validate=\"1:m\"\n",
     ")\n",
-    "joined = joined.assign(\n",
-    "    count=joined[\"count\"].fillna(0).astype(\"int64\")\n",
-    ").dropna()"
+    "joined = (joined.assign(\n",
+    "        count=joined[\"count\"].fillna(0).astype(int)\n",
+    "    ).dropna()\n",
+    "    .sort_values([\"GEOID\", \"suffix\", \"year\"])\n",
+    ")\n",
+    "\n",
+    "# Somehow, Int64 is throwing error in the map below, but int works\n",
```

Comment:
This is why I changed it to `int64` (lower case). GeoPandas still doesn't understand the `Int64` datatype.